client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -91,6 +91,11 @@
             this.stop(item);
             timeouts[item._id] = $timeout(function() {
                 var diff = extendItem({_id: item._id}, item);
+
+                if (diff.publish_schedule === null) {
```

Comment:
No this won't affect. Without this autosave was throwing exceptions '`publish_schedule` is unknown field'. When I added in the schema it was throwing exception that null is not allowed in `publish_schedule`. That's the reason I had to add this check.

However, there is one more issue with published date for which I will raise JIRA.